### PR TITLE
Add more tests for ModuleBundler

### PR DESCRIPTION
**Pull request by brendo**

### CODE REVIEW

**Comment by nfour** [comment]
@brendo conflicts ^

**Comment by brendo** [comment]
Back with you @nfour 👍

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "release": "cd dist && npm publish",
     "lint": "tslint 'src/**/*.ts'",
     "preversion": "yarn build && yarn test:dist",
-    "coverage": "jest src --coverage"
+    "coverage": "jest src --coverage --coverageReporters text text-summary"
```
Comment (nfour) [inline package.json]: Sorry, can we add this to the `jest.config.js` instead?

```
  "coverageReporters": ['text', 'text-summary'],
  "coverageThreshold": {
    "global": { lines: 50 }
  },
```

Id imagine coverage is pretty abysmal atm though

File: src/__tests__/ModuleBundler.test.ts [hunk 2]
```diff
@@ -4,28 +4,51 @@ import { Logger } from '../lib/Logger';
...
-    it(`Has bundled dependency ${dep}`, async () => {
-      expect(
-        moduleBundler.modules.some(({ name }) => name === dep),
-      ).toBeTruthy();
+    Object.keys(dependencies).forEach((dep) => {
+      expect(moduleBundler.modules.some(({ name }) => name === dep)).toBeTruthy();
     });
   });
+
+  test('can include specific dependencies', async () => {
```
Comment (nfour) [inline src/__tests__/ModuleBundler.test.ts]: Can we use `it` instead of `test`, feels more traditional with `describe`

File: package.json (hunk 1 above)
Comment (brendo) [inline package.json]: Done. And yeah, you are at 34%, so I've bumped lines down to 30% for the meantime.

File: src/__tests__/ModuleBundler.test.ts (hunk 2 above)
Comment (brendo) [inline src/__tests__/ModuleBundler.test.ts]: Updated.

**Comment by nfour** [comment]
:+1:

**Pull request merged**